### PR TITLE
FACES-2531 Suppress the rendering of component resources marked as not renderable (utilize new API for suppressing resources)

### PR DESCRIPTION
**Pull request by stiemannkj1**

### CODE REVIEW

File: bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ComponentUtil.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+/**
+ * @author  Kyle Stiemann
+ */
+public final class ComponentUtil {
+
+	private ComponentUtil() {
+		throw new AssertionError();
+	}
+
+	public static String getComponentValue(UIComponent componentResource) {
```
Comment (stiemannkj1) [inline bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ComponentUtil.java]: @ngriffin7a, this method is in `ResourceUtil`, but that doesn't really seem appropriate for it. I think we should duplicate this functionality and avoid making a `getComponentValue()` method part of the API. (This class is in an `internal` package.) When I change the API of `ResourceVerifier`, I will remove this method from `ResourceUtil` as well.

File: bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ComponentUtil.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+/**
+ * @author  Kyle Stiemann
+ */
+public final class ComponentUtil {
+
+	private ComponentUtil() {
+		throw new AssertionError();
+	}
+
+	public static String getComponentValue(UIComponent uiComponent) {
```
Comment (stiemannkj1) [inline bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ComponentUtil.java:31]: @ngriffin7a, this method is in `ResourceUtil`, but that doesn't really seem appropriate for it. I think we should duplicate this functionality and avoid making a `getComponentValue()` method part of the API. (This class is in an `internal` package.) When I change the API of `ResourceVerifier`, I will remove this method from `ResourceUtil` as well.

**Comment by ngriffin7a** [comment]
Merged into master (5.0.x) and backported to 4.0.x and 3.0.x. Thank you.

File: bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ComponentUtil.java (hunk 2 above)
Comment (ngriffin7a) [inline bridge-impl/src/main/java/com/liferay/faces/bridge/component/internal/ComponentUtil.java:31]: @stiemannkj1 Agreed, `getComponentValue()` should be removed from ResourceUtil. Please send a follow-up pull request. Thanks.

**Pull request merged**